package.json

```diff
@@ -91,7 +91,8 @@
 		"test:blackbox:infura:ws": "yarn pre-blackbox && ./scripts/verdaccio.sh startBackgroundAndPublish && lerna run test:blackbox:infura:ws --stream && yarn post-blackbox",
 		"prepare": "husky install",
 		"postinstall": "yarn build",
-		"compile:contracts": "node ./scripts/compile_contracts.js && yarn format && yarn lint:fix"
+		"compile:contracts": "node ./scripts/compile_contracts.js && yarn format && yarn lint:fix",
+		"publish:canary": "lerna --canary --dist-tag dev --graph-type all --no-verify-access --yes"
```

Comment:
Great contribution Marin, 
if we dnt increment version # this will override existing dist version in npm, or will not able to push?

Reply:
Nope, so it will publish current `[version].[commitsha]` and it will tag that version with "dev" tag. It will never generate same version twice.